integration_tests/src/gstreamer_utils.rs

```diff
@@ -0,0 +1,250 @@
+use anyhow::{anyhow, Result};
+
+use std::{
+    path::PathBuf,
+    process::{Command, Stdio},
+    thread,
+    time::Duration,
+};
+
+use super::utils::{get_asset_path, TestSample};
+
+pub fn start_gst_receive_tcp(ip: &str, port: u16, video: bool, audio: bool) -> Result<()> {
+    if !video && !audio {
+        return Err(anyhow!(
+            "At least one of: 'video_port', 'audio_port' has to be specified."
+        ));
+    }
+
+    let mut gst_output_command = [
+        "gst-launch-1.0 -v ",
+        "rtpptdemux name=demux ",
+        &format!("tcpclientsrc host={} port={} ! \"application/x-rtp-stream\" ! rtpstreamdepay ! queue ! demux. ", ip, port)
+        ].concat();
+
+    if !video && !audio {
+        return Err(anyhow!(
+            "At least one of: 'video_port', 'audio_port' has to be specified."
+        ));
+    }
```

Comment:
You already checked it at the beginning of this function. This is unreachable. Don't repeat this check.
```suggestion
```